services/paymaster/src/bin/seed.rs

```diff
@@ -118,12 +141,18 @@ pub async fn seed_from_config(config: &Config) -> Result<(), anyhow::Error> {
     if user_count.0 == 0 {
         tracing::info!("Seeding database from config");
         for domain in &config.domains {
-            let domain_url = Url::parse(&domain.domain).unwrap();
-            let host = domain_url
-                .host_str()
-                .ok_or(anyhow::anyhow!("Invalid URL"))?;
-            let user = insert_user(host).await?;
-            let app = insert_app(&user, host).await?;
+            // exception for "sessions" domain, insert that as url in the db
+            let host = if domain.domain == "sessions" {
```

Comment:
i don't think we should hardcode this, maybe we should try url parsing and fallback to using the raw domain name if url parsing fails?